code/modules/power/singularity/emitter.dm

```diff
@@ -71,7 +96,7 @@
 				src.active = 1
 				user << "<span class='notice'>You turn on \the [src].</span>"
 				src.shot_number = 0
-				src.fire_delay = 100
+				src.fire_delay = maximum_fire_delay

```

Comment:
exactly

i didn't want to touch it too much because I didn't want to be responsible for any accidental singuloth losses